### DISCUSSION TITLE
Welcome to KODI_Addons Discussions!

### DISCUSSION THREAD

**Discussion by Lunatixz** [Lunatixz/KODI_Addons#258] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Livest1** [comment]
How am I suppose to install the file downloaded from the Kodi installer add-on? Also, whenever I restart the app it'll cycle through the normal update notifications but when it comes to Kodi addon installer it says server erro

**Reply by Lunatixz** [reply]
Open Kodi installer's settings, configure your given android file explorer. After apk downloads the explorer will launch and then select the file recently downloaded and install.
